frontends/web/src/api/bitbox02.ts

```diff
@@ -77,6 +77,7 @@ export type VersionInfo = {
     // setup - that is always possible regardless of this value.
     canBackupWithRecoveryWords: boolean;
     canCreate12Words: boolean;
+    canBip85: boolean;
```

Comment:
i used canBIP85 (uppercase) in https://github.com/digitalbitbox/bitbox-wallet-app/pull/2468 :innocent: